br/pkg/conn/util/util.go

```diff
@@ -0,0 +1,62 @@
+package util
+
+import (
+	"context"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/kvproto/pkg/metapb"
+	errors2 "github.com/pingcap/tidb/br/pkg/errors"
+	"github.com/pingcap/tidb/br/pkg/version"
+	pd "github.com/tikv/pd/client"
+)
+
+// GetAllTiKVStores returns all TiKV stores registered to the PD client. The
+// stores must not be a tombstone and must never contain a label `engine=tiflash`.
+func GetAllTiKVStores(
```

Comment:
It seems it is copied from `br/pkg/conn/conn.go`. Can we keep only one `GetAllTiKVStores`?